tests/component/component.test.ts

```diff
@@ -4032,6 +4032,50 @@ describe("t-slot directive", () => {
     expect(QWeb.slots["1_footer"].toString()).toMatchSnapshot();
   });
 
+  test("named slots can define a default content", async () => {
+    env.qweb.addTemplates(`
+        <templates>
+          <div t-name="Parent">
+             <Dialog/>
+          </div>
+          <span t-name="Dialog">
+            <t t-slot="header">default content</t>
```

Comment:
maybe add the same tests **with** a slot defined, to ensure that default content is not rendered?